content/ja/docs/reference/access-authn-authz/rbac.md

```diff
@@ -0,0 +1,1109 @@
+{{< /caution >}}
+
+Kubernetesでは、Authenticatorモジュールがグループ情報を提供します。
+ユーザーと同様に、グループは文字列として表され、その文字列には、プレフィックス`system:`が予約されていることを除いて、フォーマット要件はありません。
+
+[ServiceAccounts](/docs/tasks/configure-pod-container/configure-service-account/)の名前はプレフィックス`system:serviceaccount:`が付いており、名前のプレフィックス`system:serviceaccounts:`が付いているグループに属しています。
+
+{{< note >}}
+- `system:serviceaccount:` (単数)は、サービスアカウントのユーザー名のプレフィックスです。
+- `system:serviceaccounts:`(複数)は、サービスアカウントグループのプレフィックスです。
+{{< /note >}}
+
+#### RoleBindingの例 {#role-binding-examples}
+
+次の例は`RoleBinding`、`subjects`セクションのみを示す抜粋です。
+
+`alice@example.com`という名前のユーザーの場合。
+
+```yaml
+subjects:
+- kind: User
+  name: "alice@example.com"
+  apiGroup: rbac.authorization.k8s.io
+```
+
+`frontend-admins`という名前のグループの場合。
+
+```yaml
+subjects:
+- kind: Group
+  name: "frontend-admins"
+  apiGroup: rbac.authorization.k8s.io
+```
+
+Namespace「kube-system」のデフォルトのサービスアカウントの場合。
+
+```yaml
+subjects:
+- kind: ServiceAccount
+  name: default
+  namespace: kube-system
+```
+
+Namespace「qa」の全てのサービスアカウントの場合。
+
+```yaml
+subjects:
+- kind: Group
+  name: system:serviceaccounts:qa
+  apiGroup: rbac.authorization.k8s.io
+```
+
+任意のNamespaceの全てのサービスアカウントの場合。
+
+```yaml
+subjects:
+- kind: Group
+  name: system:serviceaccounts
+  apiGroup: rbac.authorization.k8s.io
+```
+
+すべての認証済みユーザーの場合。
+
+```yaml
+subjects:
+- kind: Group
+  name: system:authenticated
+  apiGroup: rbac.authorization.k8s.io
+```
+
+認証されていないすべてのユーザーの場合。
+
+```yaml
+subjects:
+- kind: Group
+  name: system:unauthenticated
+  apiGroup: rbac.authorization.k8s.io
+```
+
+すべてのユーザーの場合。
+
+```yaml
+subjects:
+- kind: Group
+  name: system:authenticated
+  apiGroup: rbac.authorization.k8s.io
+- kind: Group
+  name: system:unauthenticated
+  apiGroup: rbac.authorization.k8s.io
+```
+
+## デフォルトRoleとClusterRoleBinding
+
+APIサーバーは、デフォルトのClusterRoleオブジェクトとClusterRoleBindingオブジェクトのセットを作成します。
+これらの多くにはプレフィックス`system:`が付いています。これは、リソースがクラスターコントロールプレーンによって直接管理されることを示しています。
+デフォルトのすべてのClusterRoleおよびClusterRoleBindingには、ラベル`kubernetes.io/bootstrapping=rbac-defaults`が付いています。
+
+{{< caution >}}
+プレフィックスとして`system:`を含む名前で、ClusterRolesおよびClusterRoleBindingsを変更する場合は注意してください。
+これらのリソースを変更すると、クラスターが機能しなくなる可能性があります。
+{{< /caution >}}
+
+### 自動調整
+
+起動するたびに、APIサーバーはデフォルトのClusterRoleを不足している権限で更新し、
+デフォルトのClusterRoleBindingを不足しているsubjectsで更新します。
+これにより、誤った変更をクラスタが修復できるようになり、新しいKubernetesリリースで権限とsubjectsが変更されても、
+RoleとRoleBindingを最新の状態に保つことができます。
+
+この調整をオプトアウトするには`rbac.authorization.kubernetes.io/autoupdate`をデフォルトのClusterRoleまたはRoleBindingのアノテーションを`false`に設定します。
+デフォルトの権限と subjectsがないと、クラスターが機能しなくなる可能性があることに注意してください。
+
+RBAC authorizerがアクティブな場合、自動調整はデフォルトで有効になっています。
+
+### APIディスカバリーRole {#discovery-roles}
+
+デフォルトのRoleBindingでは、認証されていないユーザーと認証されたユーザーが、パブリックアクセスが安全であると見なされるAPI情報(CustomResourceDefinitionsを含む)の読み取りを認可しています。匿名の非認証アクセスを無効にするには、APIサーバー構成に`--anonymous-auth=false` 追加します。
+
+`kubectl`の実行によってこれらのRoleの構成を表示するには。
+
+```shell
+kubectl get clusterroles system:discovery -o yaml
```

Comment:
(nit) I'd write:
```suggestion
kubectl get clusterrole system:discovery -o yaml
```

Reply:
Let me confirm about this.
1. I understand "kubectl get clusterroles xxx" is a full format for this and "kubectl get clusterrole xxx" also works too. Such as "kubectl get deploy" works for "kubectl get deployments", right?
2. The Original English website takes "kubectl get clusterrole**s** system:discovery -o yaml" as I kept it. Should we send to PR for original one too?

Reply:
English has plurals, Japanese doesn't. So for a language with no plurals I would use the name of the resource (lowercased) on the `kubectl get` command line.

Reply:
Thank you, I understand your point. 
I agree with you in general, but it shows [the command-line as an example](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#discovery-roles) here and I guess that we should keep the same one in both of English and Japanese, because we use same terminal and command line.

Reply:
this is fine as-is